Guess game/Program.cs

```diff
@@ -10,282 +10,98 @@ namespace Guess_game
-            //        {
-            //            Console.WriteLine("Nice you get it ");
-            //            First = First + Progress;
-            //            Console.Write("Press Enter to continue ");
-            //            Console.ReadLine();
-            //            return First;
-            //        }
-            //        else if (GuessCount == OutOfGuess)
-            //        {
-            //            Console.WriteLine("You did not get it ");
-            //            Progress = Progress - 3;
-            //            First = First + Progress;
-            //            Console.Write("Press Enter to continue ");
-            //            Console.ReadLine();
-            //            return First;
-            //        }
-            //    }
-            //}
-
-            Console.WriteLine("Welcome to riddle game you will have 3 riddles");
-            Console.WriteLine("for each riddle you will have 3 guess");
-            Console.WriteLine("Each Riddle worth 3 points" );
-            Console.WriteLine("but each wrong guess will lose 1 point");
-            Console.WriteLine("so max points in this game is 9");
-            Console.WriteLine("All riddle in Lithuanian language so answers must be in Lithuania language usin only latin letters.");
-            Console.WriteLine(" In the end we will count your result Good Luck");
-            Console.Write("Press Enter to begin ");
+            Console.WriteLine("your score is " + Result);
+            Console.WriteLine("Thanks for playing press enter to shut down program");
             Console.ReadLine();
 
-            foreach (var Riddle in Riddles)
-            {
-                Console.WriteLine(Riddle.Key, Riddle.Value);
-            }
-
-            Console.ReadLine();
+            /* Pataisiau koda metode riddle game nepriskyriau int _maxAllowedGuessCount nes kodas nedidelis ir jo uzduotis yra tik skaiciuot iki 3 tai rasiau skaiciais bet kaip sakei suprantu
+             * kodel tu taip padarei jei kazka sugalvosim perasysiu  nukopijavau tavo bool voida nes stackoverflow neradau paprastesnio o radau grynai kaip tu padarei o kaip sugalvojau butu buve
+             * du nauji metodai  todel pamaniau kam to reikia vienas false kitas true */
 
+        }
+        static int RiddleGame(int Progress)
+        {
             string begin = "y";
 
             while (begin == "y")
             {
-               
-                
-
-                Console.Write("Press 'y' to restart program  or press Enter if you want shut down program ");
-                string restartAgain = Console.ReadLine();
-
-                if (restartAgain != "y")
+                foreach (KeyValuePair<string, string> Riddle in Riddles)
                 {
-                    break;
+                    int GuessCount = 0;
+                    Console.WriteLine(Riddle.Key);
+
+                    while (GuessCount < 3)
```

Comment:
magic numbers is not good in code (now new person won't understand what is this means) i suggest to create at least variable before foreach (if you don't want to have field) and name it what does this 3 means (and same goes to 2 below, its hard to follow what code does with magic INTs)